packages/core/addon/components/cell-renderers/metric.js

```diff
@@ -28,22 +28,17 @@ export default Ember.Component.extend({
   classNames: ['table-cell-content', 'metric'],
 
   /**
-   * @property {String} metric - metric name used to fetch the value for
+   * @property {Object} attributes - metric name and optional parameters used to fetch the value for
    */
-  metric: oneWay('column.field'),
-
-  /**
-   * @property {String} format - format the number should be rendered
-   */
-  format: oneWay('column.format'),
+  attributes: oneWay('column.attributes'),
 
   /**
    * @property {Number} - value to be rendered on the cell
    */
-  metricValue: computed('data', 'metric', 'format', function() {
-    let format = get(this, 'format'),
+  metricValue: computed('data', 'attributes', function() {
+    let format = get(this, 'attributes.format'),
       type = get(this, 'column.type'),
-      canonicalName = canonicalizeMetric(get(this, 'metric')),
+      canonicalName = canonicalizeMetric(mapColumnAttributes(get(this, 'attributes'))),
```

Comment:
We do this quite a lot, makes me wonder if we should just create a method that would return a canonicalName given the column attributes.

Reply:
Yeah it would make sense having one